app/mailers/reminder_mailer.rb

```diff
@@ -1,36 +1,45 @@
 # frozen_string_literal: true
 
 class ReminderMailer < ApplicationMailer
-  def overdue_email(user, carrier_name, current_location, due_date)
-    @user = user.name.first
-    @carrier_name = carrier_name
-    @current_location = current_location
+  def overdue_email(user, carrier, current_location, due_date)
+    @user = user
+    @user_name = @user.name.first
+    @carrier_name = carrier.name
+    @current_location_name = current_location.name
```

Comment:
We are setting mostly(?) the same variables in each method. This could be DRY'd up a lot by using parameters. https://api.rubyonrails.org/classes/ActionMailer/Parameterized.html